src/rust/rust-proto/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "grapl-graph-descriptions"
+name = "rust-proto"
```

Comment:
the word `rust` feels redundant here

Reply:
Yeah, I'm not sure. `grapl-proto` doesn't feel much better. Really this is just the generated rust code for our protobuf, so I went with `rust-proto`.